libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -645,19 +638,19 @@ uint32_t AP_Frsky_Telem::calc_gps_status(void)
 {
     const AP_GPS &gps = AP::gps();
 
-    uint32_t gps_status;
+    uint32_t gps_status = 0;
```

Comment:
This is being assigned over straight away.  You could declare the variable there instead.

Reply:
ok